net/openntpd/Portfile

```diff
@@ -51,7 +51,8 @@ post-activate {
 }
 
 variant https description {Enable HTTPS Constraint Functionality} {
-    depends_lib-append      path:lib/libtls.dylib:libressl port:curl-ca-bundle
+    depends_lib-append      path:lib/libtls.dylib:libressl \
+                            path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
```

Comment:
Does this port need a revbump?

Reply:
It isn't default variant and I think that not. But it is your call and if you think that it's nessesary, I can add it. Shall I?

Reply:
Yes, please do it.

Reply:
Here it is